react/ChallengeTradePolicyCondition.tsx

```diff
@@ -92,13 +93,21 @@ const isTradePolicyAllowed = (sessionResponse: SessionResponse | undefined) => {
   }
 
   const hasAccessToTradePolicy = (sessionResponse as Session).namespaces?.store?.channel.value
+  const isLoggedIn = (sessionResponse as Session).namespaces?.profile?.email
 
-  return hasAccessToTradePolicy === "5"
+  if(isLoggedIn && hasAccessToTradePolicy === "5"){
+    return 'unauthorized'
+  } else if(isLoggedIn && hasAccessToTradePolicy !== "5"){
+    return 'authorized'
+  }
+  return 'forbidden'
```

Comment:
da pra simplificar aqui,
```suggestion
  if (!isLoggedIn) return 'forbidden'
  const isAuthorized = hasAccessToTradePolicy !== "5"
  return isAuthorized ? 'authorized' : 'unauthorized'
```